### ISSUE TITLE
not working well on heptabase V 0.3011

### ISSUE THREAD

**Issue by jellzone** [link-ding/Heptabase-Export#1]
After use, the exported content is not in pure markdown format, so the canvas behind it does not have much effect.

**Comment by link-ding** [comment]
Yes, Heptabase change their database stucture after a few days I publish this. I will write a new script to handle this.

**Comment by jellzone** [comment]
Great, thanks a lot! waits for it!
